org.eclipse.scout.rt.dataobject.test/src/test/java/org/eclipse/scout/rt/dataobject/migration/fixture/house/EntityWithIdFixtureDo.java

```diff
@@ -0,0 +1,40 @@
+/*
+ * Copyright (c) 2010, 2024 BSI Business Systems Integration AG
+ *
+ * This program and the accompanying materials are made
+ * available under the terms of the Eclipse Public License 2.0
+ * which is available at https://www.eclipse.org/legal/epl-2.0/
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ */
+package org.eclipse.scout.rt.dataobject.migration.fixture.house;
```

Comment:
Move out of .house package? Nothing to do with the other house fixtures.
Has no namespace in type name too, but probably okay for this scenario.

Reply:
Moved outside house package.
Missing namespace is similar to most of DO fixtures in org.eclipse.scout.rt.dataobject.fixture